pkg/controller/logstorage/kubecontrollers/es_kube_controllers.go

```diff
@@ -313,6 +361,7 @@ func (r *ESKubeControllersController) Reconcile(ctx context.Context, request rec
 		TrustedBundle:                trustedBundle,
 		Namespace:                    helper.InstallNamespace(),
 		BindingNamespaces:            namespaces,
+		MultiTenant:                  r.multiTenant,
```

Comment:
We should be passing `Tenant: tenant` here instead, so we can use `tenant.MultiTenant()`